omnibus/omnibus-test.sh

```diff
@@ -152,6 +152,16 @@ elif [[ -d /usr/local/etc/sudoers.d ]]; then
   sudo chmod 440 "/usr/local/etc/sudoers.d/$(id -un)-preserve_path"
 fi
 
+# Remove the ec2 cloud zypper repos before running functional tests
+if [[ ! $(grep s390 /etc/SuSE-release) ]] > /dev/null 2>&1; then   # Don't delete repos on s390x
+  if [[ $(grep -i suse /etc/SuSE-release /etc/os-release) ]] > /dev/null 2>&1; then
+     echo "Removing zypper repos for Amazon SLES nodes"
+    for i in $(zypper lr |grep -v "Alias" |awk -F "|" '{print $2}'); do
+      sudo zypper rr $i
+    done
+  fi
+fi
```

Comment:
okay this is better although we're still using "is_390x?" as a proxy for "is_not_in_ec2?" but its a lot easier to see it now...